tools/cli/commands/create.py

```diff
@@ -36,8 +44,12 @@
 _DATALAB_NETWORK = 'datalab-network'
 _DATALAB_NETWORK_DESCRIPTION = 'Network for Google Cloud Datalab instances'
 
-_DATALAB_FIREWALL_RULE = 'datalab-network-allow-ssh'
+_DATALAB_FIREWALL_RULE_TEMPLATE = '{0}-allow-ssh'
 _DATALAB_FIREWALL_RULE_DESCRIPTION = 'Allow SSH access to Datalab instances'
+_DATALAB_FIREWALLS_WARNING_TEMPLATE = ('The network `{0}` has firewall rules '
```

Comment:
Should this be FIREWALL rather than FIREWALLS? I think there is still only one firewall.